Backend/package.json

```diff
@@ -26,7 +26,7 @@
     "test:unit": "jest -c jest.config.unit.ts"
   },
   "prisma": {
-    "seed": "txs prisma/seed.ts"
+    "seed": "tsx prisma/seed.ts"
```

Comment:
why changing txs to tsx? is the previous one is not wokring?

Reply:
Yes, it was not working previously
The error that was appearing:
"An error occurred while running the seed command:
Error: Command failed with exit code 1: txs prisma/seed.ts"

What I did:
first:
 npm install tsx --save-dev
secound: 
"prisma": {
		"seed": "tsx prisma/seed.ts"
	},
The source that helped me: 
[gtihub: npx prisma db seed discussion](https://github.com/prisma/prisma/discussions/20369l)